tests/parser/markup/invalid_missing_name.todo

```diff
@@ -0,0 +1,26 @@
+{# }
+=== spans ===
+                    {# }
+SimpleMessage       ^^^^
+Markup              ^^^^
+Identifier            
+=== diagnostics ===
+[
+    Identifier is missing a name (at @2..2),
```

Comment:
Ideally we can make this message even better in a markup - like "markup is missing identifier" if both name and namespace are missing

Reply:
I left the "missing identifier" case as still panicking, since it stopped panicking by accident (that's why this test was marked as `.todo`)